curriculum/challenges/english/02-javascript-algorithms-and-data-structures/basic-javascript/using-objects-for-lookups.md

```diff
@@ -25,18 +29,22 @@ const alpha = {
   26:"A"
 };
 
-const thirdLetter = alpha[2];
-const lastLetter = alpha[24];
+const thirdLetter = reverseAlphabet[2];
+const lastLetter = reverseAlphabet[25];
 
-const value = 2;
-const valueLookup = alpha[value];
+let value = 2;
+const valueLookup = reverseAlphabet[value];
 ```
 
 `thirdLetter` is the string `Y`, `lastLetter` is the string `C`, and `valueLookup` is the string `Y`.
 
 # --instructions--
 
-Convert the switch statement into an object called `lookup`. Use it to look up `val` and assign the associated string to the `result` variable.
+Instead of using a bulky `switch` statement, let's swap this code for a more lightweight object to perform our lookup operation.
+
+Convert the switch statement into an object called `lookup` – which will store coded words from the [The International Radiotelephony Spelling Alphabet](https://en.wikipedia.org/wiki/NATO_phonetic_alphabet). 
```

Comment:
We are against the usage of external links, we are trying to make freeCodeCamp self sufficient